### PR TITLE
Template merge 110

### PR DESCRIPTION
**Pull request by Zethson**
:tada:

### CODE REVIEW

File: docs/_static/custom_cookietemple.css [hunk 1]
```diff
@@ -13,24 +13,6 @@ div, span, code {
     background-color: #005fff !important;
 }
```
Comment (Imipenem) [inline docs/_static/custom_cookietemple.css:15]: Why is all this css code removed?

File: docs/authors.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+.. include:: ../AUTHORS.rst
```
Comment (Imipenem) [inline docs/authors.rst:1]: Do we have an AUTHORS page?

File: docs/modules.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+=======
```
Comment (Imipenem) [inline docs/modules.rst:1]: Do we need this file?

File: tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+"""Unit test package for cookietemple."""
```
Comment (Imipenem) [inline tests/__init__.py:1]: Do we need this file as well?

File: docs/_static/custom_cookietemple.css (hunk 1 above)
Comment (Zethson) [reply docs/_static/custom_cookietemple.css:15]: I don't know. For some reason the cookietemple css and this one were out of sync.

Which one is the latest one?

Comment (Zethson) [reply docs/_static/custom_cookietemple.css:15]: I added the more complete one to CT AND to the common files

**Pull request merged**